ndex2/client.py

```diff
@@ -720,9 +966,39 @@ def get_task_by_id(self, task_id):
         return self.get(route)
 
     def delete_network(self, network_id, retry=5):
-        """
+        """            
+
         Deletes the specified network from the server
 
+        .. code-block:: python
+            	
+		from ndex2.nice_cx_network import NiceCXNetwork
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		# Query the network from my NDEx account, get a small subset
+		query_result_cx=my_ndex.get_neighborhood('INSERT NETWORK UUID HERE','ENTER NODE NAME HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client